service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NoticeOfProceedingsIssuedEventHandlerTest.java

```diff
@@ -54,35 +43,17 @@ void shouldNotifyAllocatedJudgeOfIssuedNoticeOfProceedingsWhenNotificationEnable
 
         CaseData caseData = caseData();
 
-        given(featureToggleService.isAllocatedJudgeNotificationEnabled(NOTICE_OF_PROCEEDINGS)).willReturn(true);
-
         given(noticeOfProceedingsEmailContentProvider.buildAllocatedJudgeNotification(caseData))
             .willReturn(expectedParameters);
 
-        noticeOfProceedingsIssuedEventHandler.notifyAllocatedJudgeOfIssuedStandardDirectionsOrder(
+        noticeOfProceedingsIssuedEventHandler.notifyAllocatedJudgeOfIssuedNoticeOfProceedings(
             new NoticeOfProceedingsIssuedEvent(caseData));
 
         verify(notificationService).sendEmail(
             NOTICE_OF_PROCEEDINGS_ISSUED_JUDGE_TEMPLATE, ALLOCATED_JUDGE_EMAIL_ADDRESS, expectedParameters,
             "12345");
     }
 
-    @Test
-    void shouldNotNotifyAllocatedJudgeOfIssuedNoticeOfProceedingsWhenNotificationDisabled() {
```

Comment:
test seems redundant as feature toggle code is actually in controller, so handler doesnt care what status of feature toggle is